fpdf/text_region.py

```diff
@@ -106,6 +142,35 @@ def write(self, text: str, link=None):
                 frag.link = link
         self._text_fragments.extend(fragments)
 
+    def generate_bullet_frags_and_tl(
+        self, bullet_string: str, rel_x_displacement: float
```

Comment:
After you changed "rel_x_displacement" to "bullet_r_margin" in the Bullet class, you should probably do the same here.